spec/lita/handlers/totems_spec.rb

```diff
@@ -142,6 +163,26 @@ def send_destroy_message
           end
         end
       end
+
+      context "with a message and a timeout" do
+        before do
+          Timecop.freeze("2014-03-01 12:00:00") do
+            send_message("totems add chicken message timeout: 10", as: carl)
```

Comment:
do we have to use it as `totems add chicken message timeout: 10`? is it possible to have the command be `totems add chicken message 10`

Reply:
If we do it that way we end up having no way to differentiate between a multiword message and a timeout, so I ended up doing it as a keyword argument sort of

Reply:
makes sense

Reply:
maybe, we can have the timeout before the message? that way we can write `totems add teal 36 some message here` and if no timeour (`totems add teal some message here`) it would default to 24